dds/DCPS/Serializer.cpp

```diff
@@ -62,36 +62,52 @@ Serializer::swapcpy(char* to, const char* from, size_t n)
   switch (n) {               // 2   4   8   16
   case 16:
     to[ 15] = from[ n - 16]; // x   x   x    0
+    // fallthrough
```

Comment:
https://gcc.gnu.org/onlinedocs/gcc-7.1.0/gcc/Warning-Options.html

It would be good to have ACE_CASE_FALLTHROUGH so it can expand to [[fallthrough]]; or one of its alternatives and it would be less likely to be removed by someone thinking it's just a comment.